config/redirects

```diff
@@ -10,4 +10,5 @@ raw: ${prefix}/stable -> ${base}/current/
 [*-master]: ${prefix}/${version}/fundamentals/crud/read-operations/watch/ -> ${base}/${version}/fundamentals/crud/read-operations/changestream/
 [*-master]: ${prefix}/${version}/usage-examples/watch/ -> ${base}/${version}/usage-examples/changestream/
 [*-master]: ${prefix}/${version}/fundamentals/crud/run-command/ -> ${base}/${version}/fundamentals/run-command/
-[*-v1.11]: ${prefix}/${version}/fundamentals/logging/ -> ${base}/${version}/
\ No newline at end of file
+[*-v1.11]: ${prefix}/${version}/fundamentals/logging/ -> ${base}/${version}/
+[*-master]: ${prefix}/${version}/fundamentals/crud/write-operations/change-a-document/ -> ${base}/${version}/fundamentals/crud/write-operations/modify/
```

Comment:
tip: when you add a redirect, paste the output of the `mut-redirects` test in the description of the PR so the reviewer can do a double check!

Reply:
Gotcha, added that in (just the output for the new redirect)